packages/store/tests/unhandled-actions.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { TestBed } from '@angular/core/testing';
+
+import { Store, NgxsModule, UnhandledActionsLogger } from '..';
+
+describe('Unhandled actions warnings', () => {
+  class FireAndForget {
+    static readonly type = 'Fire & forget';
+  }
+
+  const plainObjectAction = { type: 'Fire & Forget' };
+
+  const testSetup = (warnOnUnhandledActions = true) => {
+    TestBed.configureTestingModule({
+      imports: [NgxsModule.forRoot([], { warnOnUnhandledActions })]
+    });
```

Comment:
I propose a bit of a different API for this.
No config option, or `ignoreActions` utility, but rather a module that is provided:
```suggestion
    TestBed.configureTestingModule({
      imports: [
        NgxsModule.forRoot([]),
        ngDevMode && NgxsDevelopmentModule.forRoot({ 
          warnOnUnhandledActions: true
          /* or supply an object:
            warnOnUnhandledActions: { 
              ignore: [ 'SomeActionType', FooActionClass ] 
            }
          */
        })
      ]
    });
```
This keeps the API consolidated and clear, and follows the existing conventions.

Reply:
Love it.